spec/queries/mentoring_query_spec.rb

```diff
@@ -36,16 +36,37 @@
       let!(:brown_mentee) { create(:user, reviewer_id: brown.id) }
       let!(:other_brown_mentee) { create(:user, reviewer_id: brown.id) }
```

Comment:
does these lets need to be eager?

Reply:
Yes, because when the method is called, it search for mentees in database, if they are not created with `!`, the test will fail.

Reply:
so put this inside the context that is using it